### PR TITLE
explain oauth scopes

### PR DESCRIPTION
**Pull request by Azdaroth**

### CODE REVIEW

File: content/guides/oauth-scopes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The last one, `write owned` is not that obvious. It is still similar to `write` scope in the sense that it allows write and read actions, but the behavior is slightly different. When creating a new record with owned scope, your application will acquire a lock on the record preventing it from being modified by any other application, besides the attributes that are always whitelisted for editing (check Additional Comments section for reference). For example, when you create a booking with `bookings_write_owned` scope, only your application will be able to modify its prices'. Another difference is for `read` actions - the response might be limited for some resources, and all sensitive data for given resource will be removed (check Additional Comments section for reference). Such behavior prevents applications from accessing the sensitive data when it is not necessary for them. It is **strongly** recommended to use `write owned` scopes by default (if applicable for given resource, not all of them have the owned scope) as `write` scopes with full access might discourage property managers from installing your application.
```
Comment (StoneFrog) [inline content/guides/oauth-scopes.md]: `check Additional Comments section for reference` -> good to add link to section on `Additional Comments`

File: content/guides/oauth-scopes.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+* `expected_checkin_time`
+* `expected_checkout_time`
+* `rental_payback_to_owner`
+
+It is also possible to add new bookings fees to the locked bookings. However, editing non-owned bookings fees is not permitted.
+
+### Limited read access
+
+In case of bookings and bookings taxes, `bookings_write_owned` impacts the response body when reading the bookings that are not owned by your application and the sensitive data won't be returned. For bookings, the response body will contain only the following attributes for non-owned records:
```
Comment (StoneFrog) [inline content/guides/oauth-scopes.md]: maybe good to mention that returned attributes will be equal to the ones listed under `public` in our docs so it's easy to find out how all endpoints will behave?

File: content/reference/changelog.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Changelog
 
 ## master
-
-  * [doc update] Add guides for webhooks' subscriptions
```
Comment (StoneFrog) [inline content/reference/changelog.md]: not needed

**Comment by StoneFrog** [comment]
@Azdaroth again guide conflicts after merging last one :trollface:

**Comment by Azdaroth** [comment]
@StoneFrog updated

**Pull request merged**